packages/calypso-e2e/src/lib/base-block.ts

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
`packages/calypso-e2e` was migrated to use `import/order` a couple of weeks ago (#54542). You don't need to specify these import dock blocks anymore.

Reply:
This PR originated before the changeover, but I will make sure to remove these headers in the next revision.